mdoc/Mono.Documentation/MDocUpdater.cs

```diff
@@ -205,6 +213,15 @@ public override void Run (IEnumerable<string> args)
             { "disable-searchdir-recurse",
                 "Default behavior for adding search directories ('-L') is to recurse them and search in all subdirectories. This disables that",
                 v => DisableSearchDirectoryRecurse = true },
+            {
+                "statistics=",
+                "Save statistics to the specified file",
+                v =>
+                {
+                    statisticsEnabled = true;
+                    if (!string.IsNullOrEmpty(v))
+                        statisticsFilePath = v;
```

Comment:
I would like to see output written here, even if it's just a warning, so that the log contains the fact that the filepath is empty. Additionally, should `statisticsEnabled` be set to `false` if the filepath is null or empty (I assume yes)?

Reply:
Joel, thank you again for the detailed comments!
At the moment, as the option is declared "statistics=" passing of the parameter is required. If a user passes just "-statistics", she'll get:
```
mdoc: Missing required value for option '-statistics'.
```
So, even this check `if (!string.IsNullOrEmpty(v))` is excessive now.
We can replace `"statistics="` with `"statisitcs:"`. In this case, this option will not require a parameter. But it seems, that the current version of Mono.Options makes us pass parameters in this format:
`-statistics:some_file.txt`. If I try to pass it as `-statistics some_file.txt`, I get null instead of the passed "some_file.txt".
So, should we make it optional and add some default statistics file name?

Reply:
@mpMelnikov it's ok to make it required ... just please do some testing around the behavior of passing in: files that don't exist, files that *do* exist, folders that don't exist (ie. `-statistics some/path/file.txt`), paths that don't exist and cannot exist (ie. a fake drive letter, or path that you don't have permission to write to).

It's ok if the program can't do it, I just want it to fail gracefully, and report a reasonable error message in all cases :)